### PR TITLE
[FIX] - dbfilter_from_header: Updated the module to latest conventions.

### PR DESCRIPTION
**Pull request by fv-clearcorp**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Thanks for the followup.

:+1: fast-tracking when Travis finishes

File: dbfilter_from_header/README.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+dbfilter_from_header%0Aversion:%20
+9.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Fabio Vilchez <fabio.vilchez@clearcorp.co.cr>
```
Comment (hbrunn) [inline dbfilter_from_header/README.rst:47]: and me and @lmignon?

Comment (hbrunn) [inline dbfilter_from_header/README.rst:47]: sorry, I meant @sbidoul

File: dbfilter_from_header/README.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+server-tools/issues/new?body=module:%20
+dbfilter_from_header%0Aversion:%20
+9.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Contributors
+------------
+
```
Comment (moylop260) [inline dbfilter_from_header/README.rst:40]: Please, add old contributors

**Comment by pedrobaeza** [comment]
OK, waiting then

**Comment by fv-clearcorp** [comment]
:+1:  Done.

**Comment by moylop260** [comment]
:+1:

**Pull request merged**